### DISCUSSION TITLE
[라이프로그] 모델링 결과로 나온 해당 혼돈 행렬이 조기진단에 적합한 예측 결과라고 볼 수 있을까요 ?

### DISCUSSION THREAD

**Discussion by yonghyeun** [yonghyeun/Dementia_Analysis#7] [Ideas]
![512af677-7dd8-46e4-be09-dc9bc9a793cf](https://user-images.githubusercontent.com/123540354/235300721-a5b48283-2c75-4a0c-bfb2-b54df7164b34.png)

의료계에서 사용하는 조기 진단의 Threshold 는 조금은 엄격하고 보수적일 필요가 있다는 글을 보았습니다.

조기진단을 위해서 완전 정상군에게는 정상군임을 말해주지만 조금이라도 가능성이 보인다면 병원 혹은 보건소에서 정밀 검사를 받아볼 수 있도록 해야 할까? 하는 생각입니다...

![27dd7734-e040-4510-b940-090e53683d15](https://user-images.githubusercontent.com/123540354/235300872-b3ac932c-68f5-40a3-9586-bed64510f7ff.png)

다만 PCA를 통해 나타낸 각 class 간의 패턴을 살펴 보았을 때 각 class 간의 유의미한 차이가 있는가에 대한 생각이 들긴 하네요

feature enginearing 을 더욱 해봐야겠습니다.

**Comment by yu-hyun2** [comment]
각 class 간 차이가 없어 보이네요..있다면 개수 정도..... 혹시 저게 활동 데이터만 한 거라면 수면 데이터로 해 보는 건 어떨까요? 
수면 데이터로도 별 수확이 없으면 활동 + 수면으로 해 보는 건 어떨까요

**Reply by yonghyeun** [reply]
수면 데이터와 병합하여 해보도록 하겠습니닷
